### PR TITLE
LTP test case timeout should be logged in stderr file

### LINKED ISSUE #217: Timeouts in LTP tests not marked as such
Example: https://dev.azure.com/sgx-lkl/sgx-lkl/_build/results?buildId=219&view=ms.vss-test-web.build-test-results-tab&runId=1886&resultId=100091&paneView=attachments

This should print something to stderr if the test timed out. Currently no error is visible in the log in this case.

@hukoyu Could you have a look at that? The exit code has to be checked and if it is 124 then it timed out. Please see the equivalent logic in run_test.sh.

### PR DESCRIPTION
**Pull request by hukoyu**
This PR is addressing issue https://github.com/lsds/sgx-lkl/issues/217
We are checking exit code of sgx-lkl-run-oe and if it is 124, outputing to stderr log file.

@letmaik you can review and merge if looks good.

### CODE REVIEW

**Comment by letmaik** [comment]
Looks good, but it seems to have uncovered some timeout errors in the tests which weren't caught before?
Also, in the logs the timeout is shown like:
```
[   SGX-LKL  ] SGX-LKL (OE) Git version 9c4edff-dirty LKL version 5.3.0 [NON-RELEASE build (-O3)] [SOFTWARE DEBUG]
[   SGX-LKL  ] Warning: SGXLKL_TRACE_SIGNAL ignored in non-debug mode.
[   SGX-LKL  ] nproc=8 ETHREADS=8 CMDLINE="mem=512m" GETTIME_VDSO=0
[   SGX-LKL  ] HW TLS support: conf->fsgsbase=0
[   SGX-LKL  ] Registering disk 0 (path='sgxlkl-miniroot-fs.img', mnt='/', [RW   ])
[   SGX-LKL  ] No tap device specified, networking will not be available.
[   SGX-LKL  ] get_signed_libsgxlkl_path... result=/opt/sgx-lkl-nondebug/bin/../lib/libsgxlkl.so.signed
[   SGX-LKL  ] oe_create_enclave... result=0 (OE_OK)
[   SGX-LKL  ] sgxlkl_enclave_init(ethread_id=0)
[[  SGX-LKL ]] wg0 has public key 31mAjZ+s2cqwUmXA0Ty9cgBJLeYoKc4hQ5M4SaELVjc=
/opt/sgx-lkl-nondebug/bin/sgx-lkl-run-oe --sw-debug sgxlkl-miniroot-fs.img /ltp/testcases/kernel/syscalls/times/times03 : TIMED OUT after 60 secs. TEST_FAILED
```

Could you shorten this to:
```
TIMED OUT after 60 secs. TEST_FAILED
```
We already know which test etc is running, so this line would be enough and it's easier to find in the log.

**Comment by hukoyu** [comment]
@letmaik shortened the timeout message as you recommended. Fix commited.

**Review by letmaik — APPROVED** [review]

**Comment by letmaik** [comment]
I suggest disabling kernel-syscalls-times-times03 so that we can merge this PR. You can then have a look at why that's failing in a separate PR.

**Comment by hukoyu** [comment]
@letmaik disabled utime03 test and submitted commit.

**Comment by letmaik** [comment]
Looks like the test is still running in batch 2.

**Comment by hukoyu** [comment]
@letmaik my bad. I was supposed to disable times03, by mistake I disabled utime03. I fixed this. Disabled times03 and enabled back utime03.
